api/src/main/java/org/openmrs/module/ModuleClassLoader.java

```diff
@@ -822,10 +812,7 @@ protected File cacheLibrary(final URL libUrl, final String libname) {
 			// save a link to the cached file
 			libraryCache.put(libUri, result);
 			
-			if (log.isDebugEnabled()) {
-				log.debug("library " + libname + " successfully cached from URL " + libUrl + " and saved to local file "
-				        + result);
-			}
+			log.debug("library {} successfully cached from URL {} and saved to local file {}", libname, libUrl, result);
```

Comment:
[Here](https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging) I can see that version 1.6.x has problems when there are more than two arguments. Have you checked if the currently used version still has this problem?